internal/etl/transform/extend_values.go

```diff
@@ -134,3 +163,16 @@ func readSubstitutionsFile(filepath string) (map[string]string, error) {
 	}
 	return substitutions, err
 }
+
+func reverseMap(m map[string]string) (map[string]string, error) {
```

Comment:
In gomapgpie (en gokoala want daar komt het uit) is al een `util.Inverse()` function beschikbaar om een map om te draaien. Misschien kan je die gebruiken? Zo niet dan geen big deal, maar zou mooi zijn als het kan.

Reply:
Die had ik niet gezien, super! DONE